File: Exercicios-CS50/troco.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float troco;
    do{
        troco = get_float ("Valor do troco: ");
    } while (troco <= 0);
    int cent = round(troco*100);
    int moedas = 0;
    while(cent >= 25)
    {
        cent = cent - 25;
        moedas ++;
    }
     while(cent >= 10)
    {
        cent = cent - 10;
        moedas ++;
    }
     while(cent >= 5)
    {
        cent = cent - 5;
        moedas ++;
    }
     while(cent >= 1)
    {
        cent = cent - 1;
        moedas ++;
    }
    printf ("Total de moedas: %i", moedas);
}

File: Python/troco.py
moedas = 0
while True:
    troco = float(input('Change owed: '))
    if troco > 0:
        break
cent = round(troco*100)
while cent >= 25:
    cent = cent - 25
    moedas += 1
while cent >= 10:
    cent = cent - 10
    moedas += 1
while cent >= 5:
    cent = cent - 5
    moedas += 1
while cent >= 1:
    cent= cent - 1
    moedas += 1
print(moedas)


